packages/http/README.md

```diff
@@ -400,6 +400,12 @@ You would get this in response
 ]
 ```
 
+## Prism Decision Flow Diagram
+
+The below diagram represents all logical decision we make to figure out the best HTTP response to server. Note that this diagram is valid only when mocking is turned on.
```

Comment:
Mocking is always turned on! 

Remember, we have only the `$ prism mock` CLI command, and we have an experimental JS API we're not really advertising or pushing. Prism is a mock HTTP server and having a mock HTTP server which you need to "turn on mocking" for is going to be very confusing to the reader.

Reply:
That's not **entirely** true. I'm adding to the `http` package, not the `cli` package. To be technically valid I must say that this depends on mocking (it'll be more confusing if someone discovers that it does and that I lied that it doesn't ;p).

Reply:
The goal was for this to explain the mock server, the diagram title says Prism CLI, but your words are describing Prism HTTP which is meant to just be the client, and we're not meant to be talking about the client. 

So there's a lot of WellActually in here. Can we decide what we're trying to do and be consistent? 

If we are going to pick one thing here I would like it to be the "Prism Mock Server" and only talk about mock server functionality because we're marketing this as a mock server and the only command is mock server. :)

Reply:
Ok, kicked that confusing line out.